spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/elasticsearch/ElasticsearchRestClientProperties.java

```diff
@@ -56,6 +56,14 @@
 	 * Read timeout.
 	 */
 	private Duration readTimeout = Duration.ofSeconds(30);
+	/**
+	 * Sniffer interval
+	 */
+	private Duration sniffInterval = Duration.ofMinutes(5L);
```

Comment:
Those properties should be in a `public static class Sniffer` inner class. Check other `@ConfigurationProperties` for an example.